xbmc/filesystem/FileCache.cpp

```diff
@@ -166,10 +166,8 @@ bool CFileCache::Open(const CURL& url)
 
   CLog::Log(LOGDEBUG,"CFileCache::Open - opening <%s> using cache", url.GetFileName().c_str());
 
-  m_sourcePath = url.Get();
```

Comment:
@jhol yes that would be appreciated. It might make sense to even split them up in several if they would require additional discussion. Make sure you split them in logical commits